src/drawer.js

```diff
@@ -506,21 +499,21 @@ $.Drawer.prototype = /** @lends OpenSeadragon.Drawer.prototype */{
     },
 
     // private
-    _offsetForRotation: function( tile, degrees, useSketch ){
-        var cx = this.canvas.width / 2,
-            cy = this.canvas.height / 2;
+    _offsetForRotation: function(degrees, useSketch) {
+        var cx = this.canvas.width / 2;
+        var cy = this.canvas.height / 2;
```

Comment:
:)

About that method, is it ok to call it from tiledImage: https://github.com/openseadragon/openseadragon/pull/796/files#diff-80bee396a520b6fa750064722bc3f0ceR1335 even though it is "private"?
Or the underscore rather means "internal"?